README.md

```diff
@@ -23,7 +23,7 @@ ember install ember-frost-sort
 ```
 
 ## API
-Coming soon
+Coming soon...
```

Comment:
@sandersky Why does that matter? Particularly in a `README`? I understand it's technically more correct, but is there a reason three periods is bad? Does it break something?